astroquery/mast/tests/test_mast_remote.py

```diff
@@ -394,11 +394,26 @@ def test_observations_download_file(self, tmp_path):
 
         # pull the URI of a single product
         uri = products['dataURI'][0]
-        local_path = Path(tmp_path, Path(uri).name)
+        filename = Path(uri).name
 
-        # download it
-        result = mast.Observations.download_file(uri, local_path=local_path)
+        # download with unspecified local_path parameter
+        # should download to current working directory
+        result = mast.Observations.download_file(uri)
         assert result == ('COMPLETE', None, None)
+        assert os.path.exists(Path(os.getcwd(), filename))
+        Path.unlink(filename)  # clean up file
```

Comment:
I would prefer if everything would happen in a temporary test directory, but I suppose this is OK for now, too.

Note (though unrelated): there is still a mastDownload directory created and left on disk when running the tests on the module.

Reply:
It was a bit of a struggle, but I think I figured out the rebasing! I also edited `test_observations_download_products` so that the `mastDownload` directory is not left on the disk.

Reply:
You did indeed! 

I'll wait for a test run to finish, then will go ahead and merge. Thanks!